src/callbacks_step/amr.jl

```diff
@@ -124,6 +128,17 @@ function initialize!(cb::DiscreteCallback{Condition,Affect!}, u, t, integrator)
   amr_callback = cb.affect!
   semi = integrator.p
 
+  mesh, _, _, _ = mesh_equations_solver_cache(semi)
+
+  if mesh isa P4estMesh
+    if cb.affect!.p4est_partition_allow_for_coarsening != mesh.p4est_partition_allow_for_coarsening
+      if mpi_isroot()
+        @info "The attribute p4est_partition_allow_for_coarsening from the mesh is changed to `$(cb.affect!.p4est_partition_allow_for_coarsening)`."
```

Comment:
```suggestion
        @info "The mesh attribute `p4est_partition_allow_for_coarsening` was changed to `$(cb.affect!.p4est_partition_allow_for_coarsening)` by the `AMRCallback`."
```
I think this is less ambiguous. Also, I would print the message after changing the value (but I cannot suggest this here in the review).

Reply:
Done in 151d3bd